docs/app/docs/jetify_cloud/secrets/secrets_cli.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Managing Secrets with the Devbox CLI
+sidebar_position: 3
+---
+
+You can access your Jetify Secrets locally using Devbox. When you authenticate your Devbox CLI with Jetify Cloud, Devbox will automatically identify your project, and make your secrets available in your Devbox shell. Developers who are part of your Jetify Cloud team can also access your project’s secrets automatically, whenever they use devbox to start a shell, run a script, or launch a service
+
+If you don't already have Devbox installed, see our [Quickstart](https://www.jetify.com/devbox/docs/quickstart/) guide to get started.
+
+## Authenticating with Devbox
+
+You can authenticate with Jetify Cloud by running `devbox secrets auth login`. This will launch the browser authentication flow to sign into Jetify Cloud.
```

Comment:
```suggestion
You can authenticate with Jetify Cloud by running `devbox auth login`. This will launch the browser authentication flow to sign into Jetify Cloud.
```